src/web/app/pages-admin/admin-home-page/admin-home-page.component.ts

```diff
@@ -43,7 +66,7 @@ export class AdminHomePageComponent {
     for (const instructorDetail of this.instructorDetails.split(/\r?\n/)) {
       const instructorDetailSplit: string[] = instructorDetail.split(/[|\t]/).map((item: string) => item.trim());
       if (instructorDetailSplit.length < 3) {
-        // TODO handle error
+        // TODO handle should add all instructors when promptederror
```

Comment:
Reword to: `should handle add all instructors when prompted with error`? Was slightly confused at the comment